include/coreinit/filesystem.h

```diff
@@ -200,11 +201,13 @@ WUT_CHECK_OFFSET(FSStat, 0x24, created);
 WUT_CHECK_OFFSET(FSStat, 0x2C, modified);
 WUT_CHECK_SIZE(FSStat, 0x64);
 
-struct FSStateChangeInfo
+struct FSStateChangeParams
```

Comment:
Renaming this struct will break any existing code which uses it, so can we keep the name the same ?

Reply:
Sure maybe i'll just make a typedef out of it